3기/HyeonkyeongSeo/1주차/mission1/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
놓치기 쉬운 부분이죠! 문서 기본 언어타입을 한국어로 적용해주세요. 😊

```suggestion
<html lang="ko">
```